Source/AppAuthTV/OIDTVAuthorizationService.h

```diff
@@ -52,6 +60,30 @@ typedef void (^OIDTVAuthorizationCancelBlock)(void);
 /*! @brief Performs authorization flows designed for TVs and other limited input devices.
  */
 @interface OIDTVAuthorizationService : NSObject
+/*! @internal
+    @brief Unavailable. This class should not be initialized.
+ */
+- (instancetype)init NS_UNAVAILABLE;
+
+/*! @brief Convenience method for creating an authorization service configuration from an OpenID
```

Comment:
I think it's worth documenting why a developer would use this method, instead of the non-TV AppAuth discovery method and then calling `- (instancetype)initWithDiscoveryDocument:(OIDServiceDiscovery *)discoveryDocument {`.

That reason being (I think), that this method will perform more error checking and is designed for clients doing the TV authorization flow.

Here and below.